src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, ReactNode, useMemo } from 'react'
+import { VariantProps, tv } from 'tailwind-variants'
+
+export type Props = {
+  label: ReactNode
+  size?: VariantProps<typeof chip>['size']
+  className?: string
+}
```

Comment:
- React.CompnentProps<typeof Hoge> のような呼び出しで型は作れるので、基本的に export は不要
- label は children の方がよさそう（PropsWithChildren がそれ）
  - `<Chip label="最終評価記号" />` とするよりも
  - `<Chip>最終評価記号</Chip>` とした方がよさそう
- `size` だけ抜く必要はなさそう
- `className` 以外にも `span` が持つ属性を渡せたほうがよさそう


```suggestion
type Props = PropsWithChildren & VariantProps<typeof chip> & ComponentPropsWithoutRef<'span'>
```

Reply:
@uknmr 他コンポーネントを見ると `PropsWithChildren<Hoge>` でジェネリクスで書いてるものが圧倒的に多かったのでジェネリクスにしてみました。

FYI: TableReel.tsx でだけ `PropsWithChildren & Hoge` が使われてました